libraries/JGUZDV.Blazor.StateManagement/src/Components/StateView.razor.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.DependencyInjection;
+using System.ComponentModel;
+
+namespace JGUZDV.Blazor.StateManagement.Components;
+
+public partial class StateView<T> : StateListener
```

Comment:
Packst du den Code einfach mit in die Razor?

Reply:
Geht nicht wegen dem generischen Constraint

Reply:
https://github.com/jguzdv/NuGet-Packages/blob/8a46519b43ade7509f0040272f7858dfde4922d8/libraries/JGUZDV.Blazor.Components/src/Modals/Modal.razor#L5

Reply:
Dann war das wohl ein anderer Fehler